sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -79,6 +82,15 @@ export interface EventData {
     };
 }
 
+// @public
+export interface EventDataAdapterParameters {
+    correlationId?: string | number | Buffer;
```

Comment:
How did we choose these properties for the adapter parameters? Do you have an example of how they are used?

Reply:
They're chosen by selecting everything from the message type sane the payload and content type fields. See the [`EventData`](https://github.com/Azure/azure-sdk-for-js/blob/2c9222181416cd313d730d67ba1748ab6d94f8c2/sdk/eventhub/event-hubs/src/eventData.ts#L322-L359) type for reference.

Reply:
So this will need to be kept in sync with the EventData surface manually? Is there a way to define it once and use it in both places?

Reply:
We could take out those fields from `EventData` and make it extend this new interface instead. The only drawbacks I can think of for doing this is if the customer wants to look at the type, they'll have to hop into another place to see the rest of it. @ramya-rao-a should we expect `EventData` to gain new fields in the future? if so, how do you feel about splitting `EventData` interface into two like that?

Reply:
I would prefer to keep EventData as is and you can use the Omit magic to omit payload and content type

Reply:
That is fair. I remember that @bterlson does not like `Omit` to show up in our public APIs so I will just keep things as they're assuming that we do not expect `EventData` to evolve. I can add a comment to point out the coupling between it and this interface.

Reply:
I added a unit test that checks whether `EventDataAdapterParameters` has the same keys as `Omit<Omit<EventData, "body">, "contentType">`.

Reply:
This is the way 👍

Reply:
> I added a unit test that checks whether `EventDataAdapterParameters` has the same keys as `Omit<Omit<EventData, "body">, "contentType">`.

Nit: `Omit<Omit<EventData, "body">, "contentType">` can be expressed as `Omit<EventData, "body" | "contentType">`.